### PR TITLE
Documentation of transactionid property in v1

### PR DESCRIPTION
**Pull request by Bhupesh12Hub**
@angelgolfer-ms  Please review the PR.

Thanks

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
+| Addition | v1.0 | Added the **transactionId** property to the [event](/graph/api/resources/event?view=graph-rest-1.0) entity. |
```
Comment (angelgolfer-ms) [inline concepts/changelog.md:25]: Please move this to the September section in the changelog.

Comment (Bhupesh12Hub) [reply concepts/changelog.md]: Updated

**Comment by Bhupesh12Hub** [comment]
@angelgolfer-ms 

Changelog is updated, please review.

Thanks

**Comment by Bhupesh12Hub** [comment]
Conflict has been resolved.

**Comment by Bhupesh12Hub** [comment]
@angelgolfer-ms 

Changelog updated accordingly.

**Review by angelgolfer-ms — APPROVED** [review]
Thanks.

**Pull request merged**